src/microscope_tiling_basler_camera/src/microscope_tiling_basler_camera/main.py

```diff
@@ -0,0 +1,152 @@
+"""Example code entry point."""
+
+import os
+import numpy as np
+import cv2
+
+from numpy.typing import NDArray
+from cv2.typing import MatLike
+from zaber_motion.ascii import Connection, AxisGroup
+from zaber_motion import Measurement, Units
+from zaber_motion.microscopy import Microscope
+from basler_camera_wrapper import (
+    BaslerCameraWrapper,
+    ImageCaptureError,
+)
+from example_util import is_even, try_stitch_images, join_tiles
+from path_builder import PathBuilder
+
+# user-specified params
+SERIAL_PORT: str = "/dev/tty.usbserial-AB0PG6A5"
+PIXEL_WIDTH_MICRONS: float = 3.0177054668168553
+PIXEL_HEIGHT_MICRONS: float = 3.0177054668168553
+CAMERA_ROTATION_RAD: float = 0.045226578
+
+# TODO: reset point values to 0
```

Comment:
TODO

Reply:
Leaving this in for now so someone can run the example with the setup I've created. Will remove once that is done.

Reply:
fixed